instructions/zh.json

```diff
@@ -0,0 +1,296 @@
+{
+    "meta": {
+        "capitalizeFirstLetter": true
+    },
+    "v5": {
+        "constants": {
+            "ordinalize": {
+                "1": "第一",
+                "2": "第二",
+                "3": "第三",
+                "4": "第四",
+                "5": "第五",
+                "6": "第六",
+                "7": "第七",
+                "8": "第八",
+                "9": "第九",
+                "10": "第十"
+            },
+            "direction": {
+                "north": "北",
+                "northeast": "东北",
+                "east": "东",
+                "southeast": "东南",
+                "south": "南",
+                "southwest": "西南",
+                "west": "西",
+                "northwest": "西北"
+            },
+            "modifier": {
+                "left": "向左",
+                "right": "向右",
+                "sharp left": "向左",
+                "sharp right": "向右",
+                "slight left": "稍向左",
+                "slight right": "稍向右",
+                "straight": "直行",
+                "uturn": "调头"
+            },
+            "lanes": {
+                "xo": "靠右直行",
+                "ox": "靠左直行",
+                "xox": "保持在道路中间直行",
+                "oxo": "保持在道路两侧直行"
+            }
+        },
+        "modes": {
+            "ferry": {
+                "default": "乘坐轮渡",
+                "name": "乘坐{way_name}轮渡",
+                "destination": "乘坐开往{destination}的轮渡"
+            }
+        },
+        "arrive": {
+            "default": {
+                "default": "您已经到达您的{nth}个目的地"
+            },
+            "left": {
+                "default": "您已经到达您的{nth}个目的地, 在道路左侧"
+            },
+            "right": {
+                "default": "您已经到达您的{nth}个目的地, 在道路右侧"
+            },
+            "sharp left": {
+                "default": "您已经到达您的{nth}个目的地, 在道路左侧"
+            },
+            "sharp right": {
+                "default": "您已经到达您的{nth}个目的地, 在道路右侧"
+            },
+            "slight right": {
+                "default": "您已经到达您的{nth}个目的地, 在道路右侧"
+            },
+            "slight left": {
+                "default": "您已经到达您的{nth}个目的地, 在道路左侧"
+            },
+            "straight": {
+                "default": "您已经到达您的{nth}个目的地, 在您正前方"
+            }
+        },
+        "continue": {
+            "default": {
+                "default": "继续{modifier}",
+                "name": "继续{modifier}，上{way_name}",
```

Comment:
This seems like excessive comma usage for Chinese. Are you sure a comma is needed here?